src/status/StatusManager.cpp

```diff
@@ -34,5 +34,45 @@ namespace SlimeVR
         {
             return (m_Status & status) == status;
         }
+
+        String StatusManager::getStatusString()
+        {
+            if(m_Status == 0)
+            {
+                return "NONE";
+            }
+            String statusString;
+            if(hasStatus(Status::LOADING))
+            {
+                statusString = "LOADING";
```

Comment:
not sure if we should rather just send a bitmask from the enums 🤔

Reply:
Yeah, I changeed my mind and will return bitmask.